actioncable/lib/action_cable/connection/base.rb

```diff
@@ -95,8 +95,8 @@ def transmit(cable_message) # :nodoc:
       end
 
       # Close the WebSocket connection.
-      def close
-        websocket.close
+      def close(code = nil, reason = nil)
```

Comment:
WDYT about just delegating `close` to `websocket`?

Reply:
I don't know what it is better. But i would like to have this feature